src/main/java/org/folio/circulation/domain/policy/LostItemPolicy.java

```diff
@@ -0,0 +1,45 @@
+package org.folio.circulation.domain.policy;
+
+import io.vertx.core.json.JsonObject;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+
+public class LostItemPolicy {
```

Comment:
What makes a lost item policy different to a loan policy or overdue loan policy in this model, they all appear to have an ID and a name?

Reply:
Lost item policy and overdue loan policy models have different fields which will be added in scope of https://issues.folio.org/browse/CIRC-524 and https://issues.folio.org/browse/CIRC-548 stories.
We need only id and name for this story.

Reply:
Yeah, this is likely a flaw in the design of mod-circulation. The policy class used for references to policies within the loan should likely be different to the policies themselves.

It might be we need to perpetuate that design flaw for this change :-/